index.html

```diff
@@ -65,20 +65,20 @@
     </linkedin>
     </social_icons>
 
-    <headline>
-        <primary_text>Hey There, I'm Alex.</primary_text>
-        <supporting_text>I'm a Software Developer.</supporting_text>
-        <description>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</description>
+    <headline id="headline">
+        <primary_text id="primary_text">Hey There, I'm Alex.</primary_text>
+        <supporting_text id="supporting_text">I'm a Software Developer.</supporting_text>
```

Comment:
- [ ] Kindly I recommend you to use `h1` tag for main title and `h2` tag for sub-title , This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

Reply:
primary_text and secondary_text changed to h1 and h2 respectively as requested, font size increased as a result.